WebApp/autoreduce_webapp/templates/help.html

```diff
@@ -48,108 +48,153 @@ <h2>ISIS Compute</h2>
-                that the 'Track script changes' checkbox is ticked.
+                To make sure that reductions coming straight from the instrument use new <code>reduce.py</code>
+                scripts automatically, you can navigate to the instrument summary page fort a
+                given instrument, and select the "See instrument variables" located on the right
+                in relation ot the status table to view ongoing and upcoming instrument
+                configurations.
+                From here, you can select "edit variables" on the left side of the upcoming
+                variables table. WHen presented with the variables, ensure that the
+                "Track script changes" checkbox is checked. this is located on the right side of
+                the page. You should then select the "Submit Changes" button at the bottom of the
+                page to save this change.
             </p>
             <p>
-                When re-running a job, if you want the re-run to use the current reduce.py script click the 'Reset to current script and values' link under Additional Actions.
+                When re-running a job, if you want the re-run to use the current <code>reduce.py</code> script
+                click the "Reset to current script and values" link under Additional Actions located
+                on the right of the table.
             </p>
             <p>
-                To update instrument variables to the new ones entered into the reduce_vars.py script go to the Configure New Jobs page and click 'Reset to values in current script', this must be done any time reduce_vars.py is modified.
+                To update instrument variables to the new ones entered into the <code>reduce_vars.py</code>
+                script, navigate to the "Configure New Jobs" page from the "Instrument Summary" Page
+                and click "Reset to values in current script", located on the right side of hte page.
+                This must be done any time <code>reduce_vars.py</code> is modified.
+            </p>
+        </div>
+    </section>
+
+    <section class="help-topic panel panel-default" data-topics="script compatible work">
+        <div class="panel-heading">
+            <h4>How can I make a reduction script compatible with autoreduction?</h4>
+        </div>
+        <div class="panel-body">
+            <p>
+                For a reduction script to be compatible with the autoreduction web application
+                it must be named <code>reduce.py</code> expose what variables can be modified.
+                These variables must be made available in a file named
+                <code>reduce_vars.py</code>. For an example of this please See
+                <a href="https://github.com/ISISScientificComputing/autoreduce/wiki/Getting-Started">Here</a> for documentation.
+                and <a href="https://github.com/ISISScientificComputing/autoreduce/wiki/Example-reduction-script">Here</a>
```

Comment:
`see` and `here` to lowercase. But I'm not sure if this is intentional as other links are capitalised?

Reply:
This is intentional 👍